tests/managers/opportunity-test.js

```diff
@@ -111,6 +112,14 @@ describe('OpportunityManager', function () {
         queryObject,
         updatingobject
       );
+
```

Comment:
@Kashish0423 Test the API on postman yourself, see what it does when 
1. 12-bit incorrect id is passed
2. lesser or greater than 12 bit id is passed

See the API Response in both cases and write the test cases accordingly. 

Also, the test `updateOpportunity` has to be updated with more test cases, you wont have multiple `describe blocks`, instead multiple `it` blocks. 
In `tests/managers/opportunity-test.js`, `tests/services/opportunity-test.js` , `tests/controllers/opportunity-test.js`, the test `describe('updateOpportunity` is to be updated with more test cases i.e more `it` blocks.

Reply:
Okay mam  on it , I will try to get it done asap